src/zarr/buffer/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+Public API for implementations of the Zarr Buffer interface.
+
+See Also
+========
+arr.abc.buffer: Abstract base class for the Zarr Buffer interface.
```

Comment:
```suggestion
:mod:`arr.abc.buffer` : Abstract base class for the Zarr Buffer interface.
```
Would be good to give this a link

Reply:
I confirmed locally that numpydoc / sphinx do this automatically.